e2e/stack_test.go

```diff
@@ -231,6 +234,37 @@ func TestE2E(t *testing.T) {
 	t.Log("Monomer blocks contain the l1 attributes deposit tx")
 }
 
+func requireEthIsMinted(t *testing.T, appchainClient *bftclient.HTTP) {
+	query := "tx.height > 0"
```

Comment:
I think we can search for the specific event. The transaction indexer [indexes the events](https://docs.cometbft.com/v0.38/app-dev/indexing-transactions#adding-events) iirc.

Reply:
It does, but I burned nontrivial time trying to execute a more precise query.

For expediency, I'm inclined to leave this brutish check, which is still pretty readable.

Reply:
SGTM. Mind creating a "good first issue" for this? Would be a good educational exercise for new joiners.

Reply:
Feel free to merge this PR whenever.